### PR TITLE
Tutorial removing fields

### PR DESCRIPTION
**Pull request by tanruiquan**
Remove `Address` because such information is not important to a TA

### CODE REVIEW

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 1]
```diff
@@ -34,7 +32,6 @@ public PersonBuilder() {
         name = new Name(DEFAULT_NAME);
         phone = new Phone(DEFAULT_PHONE);
         email = new Email(DEFAULT_EMAIL);
-        address = new Address(DEFAULT_ADDRESS);
```
Comment (James-Kua) [inline src/test/java/seedu/address/testutil/PersonBuilder.java:37]: Can we remove line 21 `DEFAULT_ADDRESS` as well?

**Review by James-Kua — COMMENTED** [review]
Just one bit of Address trace left.

**Review by czhi-bin — COMMENTED** [review]
LGTM 👍

**Review by driip-ddrop — COMMENTED** [review]
lgtm :)

**Review by James-Kua — APPROVED** [review]
LGTM!

**Pull request merged**